code/modules/mob/living/carbon/human/human_powers.dm

```
@@ -270,6 +270,38 @@
 			else if(prob(50))
 				to_chat(H,"<span class='warning'>Your mind buzzes...</span>")
 
+/mob/living/carbon/human/proc/swallowobject()
+	set name = "Swallow Object"
+	set desc = "Swallows a held object"
+	set category = "IC"
+
+	var/mob/living/carbon/human/H = src
+	var/obj/item/organ/ST = H.internal_organs_by_name["stomache"]
+	var/obj/item/organ/external/E = H.organs_by_name[ST.parent_organ]
+	var/obj/item/I = H.get_active_hand()
+
+	if(!I)
+		to_chat(H, "<span class='notice'>You can't swallow nothing!</span>")
```

```suggestion
```

---

Just have this be a silent failure.